smoothscroll.js

```diff
@@ -76,20 +76,35 @@
   }
 
   /*
+   * checks if element can scroll. 
+   * NOTE: 5 px buffer added because scrollHeight = clientHeight + padding.
+   * MDN: "It includes the element padding but not its margin."
+   * https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollHeight
+   * Too much overhead to check for padding? => Change the "5" to whatever padding you're using.
```

Comment:
since this is a polyfill accuracy is more important. Checking padding is going to be marginally more costly than checking clientHeight and scrollHeight already (since we’re already triggering layout).